src/js/outreach/apps/verify_app.js

```diff
@@ -20,6 +20,14 @@ export default App.extend({
   beforeStart({ actionId }) {
     return getPatientInfo({ actionId });
   },
+  onFail() {
+    const dialogView = new DialogView();
+    this.showView(dialogView);
+
+    this.showNotAvailableView();
+
+    // this.stop();
```

Comment:
Do we need the `this.stop()`? Other `onFail`s in the app have an `this.stop()`.

If I call `this.stop()` here, it starts the form app via this: https://github.com/RoundingWell/care-ops-frontend/blob/develop/src/js/outreach/index.js#L46-L49.

Reply:
No need to stop, the app is showing a view `onFail`